app/src/operations/notification/index.ts

```diff
@@ -19,13 +19,13 @@ export default defineOperationApp({
 		{
 			field: 'recipient',
 			name: '$t:operations.notification.recipient',
-			type: 'string',
+			type: 'json',
```

Comment:
And now for the fun question: will the `tags` interface break when `json` is used but an (old) single string value is passed? 😄

Reply:
Just tested it and it doesn't seem to break as the fix for tags interface in the raw editor PR does prevent it from breaking 😄

That being said, you did make me fully realize the change to json was never included with tags... 🤦‍♂️ 

Currently all the tags interface fields across all Flow operations are actually using `csv`, with one exception being `To` in Email operation which is still using `string`:

https://github.com/directus/directus/blob/080f729079e5e998e8d651b06dcdbfb4fd6d8c5d/app/src/operations/mail/index.ts#L24-L29

Should we make them all csv for uniformity instead?

Reply:
> Currently all the tags interface fields across all Flow operations are actually using csv, with one exception being To in Email operation which is still using string:

Oh! In that case, lets use `csv` everywhere yeah 👍🏻

Reply:
Done 👍